blobfuse/OAuthTokenCredentialManager.cpp

```diff
@@ -99,15 +108,21 @@ OAuthToken OAuthTokenCredentialManager::get_token()
         if (token_mutex.try_lock()) {
             try {
                 // Attempt to refresh.
+                fprintf(stdout, "oauth token has expired so calling refresh_token()\n");
+                syslog(LOG_WARNING, "oauth token has expired so calling refresh_token()\n");
+          
                 refresh_token();
             } catch (std::runtime_error &ex) {
                 // If we fail, explain ourselves and unlock.
-                syslog(LOG_ERR, "Unable to retrieve OAuth token: %s", ex.what());
+                syslog(LOG_ERR, "Unable to retrieve OAuth token: %s\n", ex.what());
                 valid_authentication = false;
                 token_mutex.unlock();
             }
         } else {
+            fprintf(stdout, "Locking mutex failed, so some token is being acquired., so just wait and get that\n");
+            syslog(LOG_WARNING, "Locking mutex failed, so some token is being acquired., so just wait and get that\n");

```

Comment:
Update string to be "Currently attempting to refresh token, initiating delay"

Reply:
leave it, it is heavy dev information.